<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compras F치cil</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Imagens.qrc">
    <normaloff>:/imagens/icones/icon.png</normaloff>:/imagens/icones/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelCompras">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Compras</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyTreeView" name="treeViewCompras">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>false</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelProdutos">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Produtos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MyTreeView" name="treeViewProdutos">
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="actionEdit"/>
   <addaction name="actionSalvar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuFerramentas">
    <property name="title">
     <string>Relat칩rios</string>
    </property>
    <addaction name="actionGerar_Relatorio_Mensal"/>
    <addaction name="actionGerar_Relat_rio_por_Classe_de_Produto"/>
    <addaction name="actionGerar_Relat_rio_Mensal_Classe"/>
    <addaction name="actionMaiores_Gastos"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuEditar"/>
   <addaction name="menuFerramentas"/>
  </widget>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="Imagens.qrc">
     <normaloff>:/imagens/icones/add.png</normaloff>:/imagens/icones/add.png</iconset>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="Imagens.qrc">
     <normaloff>:/imagens/icones/remove.png</normaloff>:/imagens/icones/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remover</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="Imagens.qrc">
     <normaloff>:/imagens/icones/edit.png</normaloff>:/imagens/icones/edit.png</iconset>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="icon">
    <iconset resource="Imagens.qrc">
     <normaloff>:/imagens/icones/salvar.png</normaloff>:/imagens/icones/salvar.png</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionGerar_Relatorio_Mensal">
   <property name="text">
    <string>Relatorio Mensal</string>
   </property>
  </action>
  <action name="actionGerar_Relat_rio_por_Classe_de_Produto">
   <property name="text">
    <string>Relat칩rio por Classe de Produto</string>
   </property>
  </action>
  <action name="actionGerar_Relat_rio_Mensal_Classe">
   <property name="text">
    <string>Relat칩rio Mensal + Classe</string>
   </property>
  </action>
  <action name="actionMaiores_Gastos">
   <property name="text">
    <string>Maiores Gastos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">mytreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Imagens.qrc"/>
 </resources>
 <connections/>
</ui>
